### PR TITLE
v9 - v10 Upgrade step improvements

### PR DESCRIPTION
**Pull request by JasonElkin**
I tried to follow the Upgrade steps 1 to 8 verbatim but step 8 doesn't actually work until after the old files have been deleted.

I've updated the steps to include the Program.cs change and file deletion to make it clear these are a necessary part of the process. 

Also, I'm pretty sure that the whole `/umbraco/config` directory can go.

### CODE REVIEW

File: Fundamentals/Setup/Upgrading/version-specific.md [hunk 1]
```diff
@@ -68,21 +66,20 @@ public class Program
...
+    ```
+    The calls to `ConfigureUmbracoDefaults` and `webBuilder.UseStaticWebAssets()` are new.
+
+9. Remove the following files and folders:
+    - `/wwwroot/umbraco`
+    - `/umbraco/PartialViewMacros`
+    - `/umbraco/UmbracoBackOffice`
+    - `/umbraco/UmbracoInstall`
+    - `/umbraco/UmbracoWebsite`
+    - `/umbraco/config`
```
Comment (eshanrnh) [inline Fundamentals/Setup/Upgrading/version-specific.md:78]:
```suggestion
    - `/umbraco/config/lang`
```

**Review by eshanrnh — APPROVED** [review]
Looks great, @JasonElkin 🙌 Thanks for rearranging and formaating the steps 💪

Regarding the `/umbraco/config` directory, we need to only delete  the /umbraco/config/lang folder as there maybe some packages and other stuff in that folder which we do not want the users to delete🙂

**Review by eshanrnh — CHANGES_REQUESTED** [review]
Looks great, @JasonElkin 🙌 Thanks for rearranging and formatting the steps 💪

Regarding the `/umbraco/config` directory, we need to only delete the `/umbraco/config/lang` folder as there may be some packages and other stuff in that folder that we do not want the users to delete🙂

File: Fundamentals/Setup/Upgrading/version-specific.md [hunk 2]
```diff
@@ -68,21 +66,20 @@ public class Program
...
+    - `/App_Plugins/UmbracoForms` (if you are using Umbraco Forms on your project)
+10. Build and run your project to finish the installation of Umbraco 10.
+
+To re-enable the appsettings IntelliSense, you must update your schema reference in the `appsettings.json` file, and any other `appsettings.{Environment}.json` files from:
```
Comment (eshanrnh) [inline Fundamentals/Setup/Upgrading/version-specific.md:82]:
```suggestion
To re-enable the appsettings IntelliSense, you must update your schema reference in the `appsettings.json` file and any other `appsettings.{Environment}.json` files from:
```

File: Fundamentals/Setup/Upgrading/version-specific.md (hunk 1 above)
Comment (JasonElkin) [reply Fundamentals/Setup/Upgrading/version-specific.md:78]: OK, in that case I think we should also remove the old appsettings schema specifically.

**Review by eshanrnh — APPROVED** [review]
That's a good point 👏 The changes look good to me 💪

Merging it in.

**Comment by eshanrnh** [comment]
Also, I can see this is the first time you're contributing to the Umbraco Documentation 🎉 That's awesome!! Hope to see you back with more contributions in the future 😎
Do you have an account on [Our](https://our.umbraco.com/)? I can grant you a shiny new C-trib badge for your efforts, so everyone can see you're helping out 😄 Let me know!

**Comment by JasonElkin** [comment]
Thanks @eshanrnh, happy to help and already got my C-trib badge!

**Pull request merged**